helm-service/controller/helm/generated_chart_handler.go

```diff
@@ -220,8 +230,10 @@ func (c *GeneratedChartHandler) generateServices(svc *corev1.Service, project st
 
 	// Generate virtual service
 	gws := []string{GetGatewayName(project, stageName) + "." + GetUmbrellaNamespace(project, stageName), "mesh"}
-	hosts := []string{svc.Name + "." + c.canaryLevelGen.GetNamespace(project, stageName, false) + "." + c.keptnDomain,
-		svc.Name, svc.Name + "." + c.canaryLevelGen.GetNamespace(project, stageName, false)}
+	hosts := []string{
```

Comment:
Please add a short comment here behind each line of this array, showing an example of what the host looks like:
* e.g., `service_name.dev.123.45.67.89.xip.io`
* e.g., `service_name`